t/unittest/UnifiedApi/Plugin/RestCrud.t

```diff
@@ -1,490 +0,0 @@
-#!/usr/bin/perl
```

Comment:
Looks like this is gone but not replaced by anything else.

Anyway we can keep the same level of testing ?

Reply:
Same level of testing is not needed.

The plugin only provides very simple shortcuts to the route class.
If the shortcuts did not work then routing would not work.